test/test_nestedtensor.py

```diff
@@ -6589,10 +6599,24 @@ def test_sdpa(self, device, dtype):
         )
         self.assertEqual(attn_out.shape, q_nt_3.shape)
 
-        def check_forward_backward():
-            attn_nt = torch.nn.functional.scaled_dot_product_attention(
-                q_nt_t, k_nt_t, v_nt_t
-            ).transpose(1, 2)
+        # TODO(eqy): skip_backward can be removed when cuDNN NT backward is added
+        def check_forward_backward(skip_backward=False):
```

Comment:
tiny nit: you could alternatively parametrize the test on whether to do backward. this allows an xfail when the cuDNN backend is in use

Reply:
`decorateIf` should give you this btw: https://github.com/pytorch/pytorch/blob/6ff3383157b9a9d0342822639919683ded827483/torch/testing/_internal/common_utils.py#L777-L812

Reply:
Sure, I added an `assertRaises` which tries to exercise the test.